src/docs/asciidoc/15-examples.adoc

```diff
@@ -30,6 +30,52 @@ task buildImage(type: DockerBuildImage) {
 }
 ----
 
+==== Interacting With a Container Running in the Background
+
+This code demonstrates managing a container running in the background with this plugin
+which can facilitate locally running a Dockerized application. This can be done by
+creating the container with an explicit `containerName` which is then used as
+the `targetContainerId` for relevant tasks.
+
+.build.gradle
+[source,groovy]
+----
+apply plugin: 'com.bmuschko.docker-remote-api'
+
+import com.bmuschko.gradle.docker.tasks.container.*
+
+ext {
+  CONTAINER_NAME = 'myapp'
```

Comment:
Extra properties are not really environment variables. Could you please use camel case notation here and use the 4 spaces indentation?